### PR TITLE
Converts artwork page to palette v3

### PR DESCRIPTION
**Pull request by dzucconi**
https://user-images.githubusercontent.com/112297/120709545-f7fcc180-c48a-11eb-9b9d-625c0caca7fe.mov

### Post Launch
- [ ] Better use space for primary artwork image (optimize images for device)
- [ ] Appropriately sized placeholders for images (lazy loading)
- [ ] Rebuild view in room
- [ ] Rebuild lightbox

### CODE REVIEW

**Comment by dzucconi** [comment]
This is ready but I'll get a review app of this deployed.

File: src/v2/Apps/Artwork/Components/ArtworkImageBrowser/ArtworkActions.tsx [hunk 1]
```diff
@@ -330,117 +331,101 @@ interface UtilButtonProps {
...
-            )}
-          </>
-        )}
-      </UtilButtonContainer>
-    )
-  }
+        {/* TODO: Fix types */}
+        {/* @ts-ignore */}
+
+        <ActionIcon {...rest} fill="currentColor" />
```
Comment (dzucconi) [inline src/v2/Apps/Artwork/Components/ArtworkImageBrowser/ArtworkActions.tsx:389]: I really need to add `currentColor` to the icon fill typings.

File: src/v2/Apps/Artwork/Components/ArtworkImageBrowser/ArtworkImageBrowser.tsx [hunk 2]
```diff
@@ -1,239 +1,197 @@
 import { ArtworkImageBrowser_artwork } from "v2/__generated__/ArtworkImageBrowser_artwork.graphql"
-import { BaseCarousel as Carousel } from "v2/Components/FlickityCarousel"
```
Comment (dzucconi) [inline src/v2/Apps/Artwork/Components/ArtworkImageBrowser/ArtworkImageBrowser.tsx:2]: 👋  byeeeee

File: src/v2/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCommercial.tsx [hunk 3]
```diff
@@ -34,6 +34,9 @@ import { openAuthModal } from "v2/Utils/openAuthModal"
 import { ArtworkSidebarSizeInfoFragmentContainer as SizeInfo } from "./ArtworkSidebarSizeInfo"
 import { Mediator } from "lib/mediator"
 
+// TODO:
+// - Check every variation...
```
Comment (dzucconi) [inline src/v2/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCommercial.tsx:38]: Sorry, I gotta remove all these note-to-self TODOs.

**Review by damassi — APPROVED** [review]
Its a big one, so i'll just trust ya...

**Comment by dzucconi** [comment]
OK. This should be green now and I feel confident but I'm not about to merge this at 6pm on a Friday. So: first thing Monday 🚀

**Pull request merged**